### PR TITLE
regression: add case 1002

### PR DESCRIPTION
**Pull request by jenswi-linaro**
Adds regression case 1002 to test passing a memory buffer to a Pseudo
TA.

Signed-off-by: Jens Wiklander <jens.wiklander@linaro.org>

Depends on https://github.com/OP-TEE/optee_os/pull/1884

### CODE REVIEW

File: host/xtest/regression_1000.c [hunk 1]
```diff
@@ -288,6 +290,46 @@ static void xtest_tee_test_1001(ADBG_Case_t *c)
...
+	TEEC_Operation op = TEEC_OPERATION_INITIALIZER;
+	uint32_t ret_orig;
+	uint8_t buf[16 * 1024];
+	uint8_t exp_sum = 0;
+	size_t n;
+
+	res = xtest_teec_open_session(&session, &pta_invoke_tests_ta_uuid, NULL,
+				      &ret_orig);
+	/*
+	 * If the static TA (which is optional) isn't available, skip this
```
Comment (jforissier) [inline host/xtest/regression_1000.c]: s/static/pseudo/

**Comment by jenswi-linaro** [comment]
Comment addressed, tag applied

**Comment by etienne-lms** [comment]
`Reviewed-by: Etienne Carriere <etienne.carriere@linaro.org>`

**Comment by jforissier** [comment]
Etienne's R-b added, merged.

**Pull request closed without merging**